typeConsts.go

```diff
@@ -1,4 +1,4 @@
-// SPDX-FileCopyrightText: 2023 SAP SE
+// SPDX-FileCopyrightText: 2024 SAP SE
```

Comment:
Why does it replace the 2023 entry here while it adds the 2024 entry everywhere else?